tests/vertical_playground.html

```diff
@@ -396,6 +399,54 @@
           </value>
         </block>
       </category>
+      <category name="Pen">
+        <block type="pen_clear"></block>
+        <block type="pen_stamp"></block>
+        <block type="pen_penup"></block>
+        <block type="pen_pendown"></block>
+        <block type="pen_change_pen_color_by">
+          <value name="COLOR">
+            <shadow type="math_number">
+              <field name="COLOR">10</field>
```

Comment:
All of these fields inside the `math_number` shadows should be `<field name="NUM">`, I think.